From 828feae4f1814a915b2f7f362a5920322e0d6fcc Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 9 Dec 2011 10:28:40 +0100
Subject: [PATCH] dracut.spec: remove unnecessary dependencies

Since the initramfs generation is done in %postrans of the kernel rpm,
we can drop all hard requirements.

Also make some requirements a conflict to express the version
dependency.
---
 dracut.spec |   24 ++----------------------
 1 files changed, 2 insertions(+), 22 deletions(-)

diff --git a/dracut.spec b/dracut.spec
index 91b62ee..1c50f37 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -68,17 +68,15 @@ Requires: filesystem >= 2.1.0
 Requires: findutils
 Requires: grep
 Requires: gzip
-Requires: kbd
 Requires: mktemp >= 1.5-5
 Requires: module-init-tools >= 3.7-9
 Requires: sed
-Requires: tar
 Requires: udev
 Requires: util-linux >= 2.20
 
 %if 0%{?fedora} || 0%{?rhel} > 6
-Requires: initscripts >= 8.63-1
-Requires: plymouth >= 0.8.0-0.2009.29.09.19.1
+Conflicts: initscripts < 8.63-1
+Conflicts: plymouth < 0.8.0-0.2009.29.09.19.1
 %endif
 
 %description
@@ -91,24 +89,6 @@ NFS, iSCSI, NBD, FCoE with the dracut-network package.
 %package network
 Summary: Dracut modules to build a dracut initramfs with network support
 Requires: %{name} = %{version}-%{release}
-Requires: rpcbind
-%if %{with_nbd}
-Requires: nbd
-%endif
-Requires: iproute
-Requires: bridge-utils
-
-%if 0%{?fedora} || 0%{?rhel} > 6
-Requires: iscsi-initiator-utils
-Requires: nfs-utils
-Requires: dhclient
-%endif
-
-%if 0%{?suse_version}
-Requires: dhcp-client
-Requires: nfs-client
-Requires: vlan
-%endif
 Obsoletes: dracut-generic < 008
 Provides:  dracut-generic = %{version}-%{release}
 
